### ISSUE TITLE
Update syntax color to make them richer

### ISSUE THREAD

**Issue by octref** [arcticicestudio/nord-visual-studio-code#12]
First, thanks a lot for the workbench update. I'm using it and it's awesome.

However, I hope the syntax can be updated a bit to match the richness of Dark+'s syntax highlighting. When I'm working on TS, I rely on colors as visual cues. For example, Type information is always marked as Green.

Here is a pic of Dark+ vs Nord.

![slice](https://cloud.githubusercontent.com/assets/4033249/25783328/78da4834-3328-11e7-8b71-a1bf22ffa662.jpg)

**Comment by Livven** [comment]
Completely agree. Ideally maybe the syntax colors could be simply based on the default (dark+/light+) themes, so that each color value from the default theme is replaced with the same color value every time. This way you'd get exactly the same behavior and "richness" as the default theme, only with a different aesthetic.

Obviously for the workbench you might want to get a bit more creative (as in having the same background color for all three panels which may not be possible with the above method) but for syntax colors I don't think this kind of customization is necessary.

**Comment by pomadin** [comment]
@arcticicestudio Hi, indeed the theme is pretty cool. It seems that there are some mismatchings (misprints?) in scope names. For instance
```js
      "name": "Entity Name Class/Type",
      "scope": [
        "entity.name.class",
        "entity.name.type.class" // should it be: entity.name.type?
        ],
```

**Comment by arcticicestudio** [comment]
@pomadin I've already thought about checking all scopes since the described issues worked before I've migrated to the new theming API. I've used their builtin migration tool which converted the TextMate theme automatically into the JSON format, but it may be that there have been some converting problems.
I'll try to check all scopes next weekend.

**Comment by Livven** [comment]
@arcticicestudio Pretty sure TS types not being highlighted was an issue in the previous version (before the workbench update) already, before the switch to the JSON format.

**Comment by octref** [comment]
Don't know if you are aware, but Cmd+Shift+P -> Inspect TM Scopes helps a lot for diagnosing.

![image](https://cloud.githubusercontent.com/assets/4033249/26026749/63fbc8b0-37cf-11e7-8bba-a48e99316698.png)

![image](https://cloud.githubusercontent.com/assets/4033249/26026757/8c14dba2-37cf-11e7-8746-8640de766c9d.png)

Also, take a look at this folder: https://github.com/Microsoft/vscode/tree/master/extensions/theme-defaults/themes

`dark_plus` includes `dark_vs`, which includes `dark_default`.
I believe Nord already covers `dark_vs`, so you just need to have those in `dark_plus` added to the theme, like such:

![image](https://cloud.githubusercontent.com/assets/4033249/26026777/0fc9c82c-37d0-11e7-8b11-462942b6412e.png)

**Comment by octref** [comment]
I can do a PR if you are ok with it. I really want to use Nord as my daily driver but this lack of functionality makes it unusable for me :-(

**Comment by arcticicestudio** [comment]
Sorry for the late reply, I'm really busy the whole week long. I hope to get some time this weekend. Anyway, feel free to open a PR, I'm always thankful for any help 😄 That would also take me a lot of work to write a script compare the differences of the JSON keys to find missing scopes 😆

**Comment by Ikuyadeu** [comment]
I think these need variable
For example vscode dark+

Variable
```
		{
			"name": "Variable and parameter name",
			"scope": [
				"variable",
				"meta.definition.variable.name",
				"support.variable"
			],
			"settings": {
				"foreground": "#9CDCFE"
			}
		},
```

**Comment by arcticicestudio** [comment]
Sorry for the late response, haven't got time for private projects the last months.
I've pushed some changes regarding missing TypeScript syntax highlights and some CSS improvements.

**Comment by octref** [comment]
Thanks a lot! Looking forward to 0.4.0 release.

**Comment by arcticicestudio** [comment]
🚢 Shipped with extension release version 🏷 [0.4.0](https://github.com/arcticicestudio/nord-visual-studio-code/releases/tag/v0.4.0).
